js/client/modules/@arangodb/testutils/testrunner.js

```diff
@@ -304,7 +304,7 @@ let failurePointsCheck = {
   name: 'failurepoints',
   setUp: function(obj, te) { return true; },
   runCheck: function(obj, te) {
-    let failurePoints = pu.checkServerFailurePoints(obj.instanceManager);
+    let failurePoints = obj.instanceManager.checkServerFailurePoints(obj.instanceManager);
```

Comment:
Why `obj.instanceManager.checkServerFailurePoints(obj.instanceManager)` can't just pick itself (`obj.instanceManager`) by default?

Reply:
the function may work with an instanceManager that has been de/serialized via the OS-environment

Reply:
Yes. But by default (empty value is passed) it can pick `this`.